aten/src/ATen/native/cuda/ForeachUnaryOp.cu

```diff
@@ -85,7 +85,7 @@ void foreach_unary_op_complex_bfloat16_(TensorList tensors) {
 }
 
 template <template<class> class Op>
-std::vector<Tensor> foreach_unary_op(TensorList tensors) {
+std::vector<Tensor> all_types_half_bfloat16(TensorList tensors) {
```

Comment:
This is new, right? Does this PR add additional behavior to any of the foreach ops? If so we should note it in the PR description and add tests for those behaviors